Plugin/OrderPlugin.php

```diff
@@ -69,6 +90,12 @@ public function beforeSetStatus(Order $subject, $status)
         if (!$subject->getPayment() || $subject->getPayment()->getMethod() != \Bolt\Boltpay\Model\Payment::METHOD_CODE) {
             return [$status];
         }
+
+        if ($subject->getIsRechargedOrder()) {
+            // Check and set the recharged order state to processing
```

Comment:
Could you add a comment while we need set status to processing in this case? What happens if we don't do this?